src/test/java/com/wikia/webdriver/testcases/discussions/CategoriesTests.java

```diff
@@ -473,12 +473,12 @@ private void openPageAndSelectCategoryOnMobile(PostsListPage page, String catego
     final FiltersPopOver filtersPopOver = page.getFiltersPopOver().click();
     filtersPopOver.getCategoriesFieldset().clickCategoryWith(categoryName);
     filtersPopOver.clickApplyButton();
-    page.waitForLoadingSpinnerWith(categoryName);
+    page.waitForPageReloadWith(categoryName);
   }
 
   private void openPageAndSelectCategoryOnDesktop(PostsListPage page, String categoryName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openPageAndSelectCategoryOnDesktop" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)